### PR TITLE
feature/PPM-5 Integration - Client BML APIs

### PR DESCRIPTION
**Pull request by itayx**
Added small fixes for the smart steering BML client API
- In client_set_client changed incoming type from INT_ARG to STRING_ARG.
- Added int conversion in the client_set_client LOG print.
- Added tlvf::mac_to_string in the client_get_client LOG print.

### CODE REVIEW

File: controller/src/beerocks/bml/bml_defs.h [hunk 1]
```diff
@@ -599,7 +599,7 @@ struct BML_CLIENT_CONFIG {
 
 struct BML_CLIENT {
     // Client MAC.
-    char sta_mac[BML_MAC_ADDR_LEN];
+    uint8_t sta_mac[BML_MAC_ADDR_LEN];
```
Comment (adam1985d) [inline controller/src/beerocks/bml/bml_defs.h:602]: commit **title** should be:
`bml: BML_CLIENT: change sta_mac type to unit8_t`
or
`bml: change sta_mac type in BML_CLIENT to unit8_t`

1. bml not bcl
2. current title doesn't say anything or doesn't say enough

File: controller/src/beerocks/cli/beerocks_cli_bml.cpp [hunk 2]
```diff
@@ -563,8 +563,8 @@ void cli_bml::setFunctionsMapAndArray()
         " selected_bands - Bitwise parameter, 1 for 2.4G, 2 for 5G, 3 for both, 0 for Disabled"
         " stay_on_initial_radio - 1 for true, 0 for false or (default) -1 for not configured,"
         " stay_on_selected_device - 1 for true, 0 for false or (default) -1 for not configured",
-        static_cast<pFunction>(&cli_bml::client_set_client_caller), 2, 4, STRING_ARG, INT_ARG,
-        INT_ARG, INT_ARG);
+        static_cast<pFunction>(&cli_bml::client_set_client_caller), 2, 4, STRING_ARG, STRING_ARG,
```
Comment (adam1985d) [inline controller/src/beerocks/cli/beerocks_cli_bml.cpp:566]: 1. commit message:
- change `beerocks_cli` to `beerocks_cli_bml`
- please explain on each change why the changes are needed

File: controller/src/beerocks/cli/beerocks_cli_bml.cpp [hunk 3]
```diff
@@ -2230,9 +2230,10 @@ int cli_bml::client_set_client(const std::string &sta_mac, int8_t selected_bands
 {
     std::cout << "client_set_client: " << std::endl
               << "  sta_mac: " << sta_mac << std::endl
-              << "  selected_bands: " << selected_bands << std::endl
-              << "  stay_on_initial_radio: " << stay_on_initial_radio << std::endl
-              << "  stay_on_selected_device: " << stay_on_selected_device << std::endl;
+              << "  selected_bands: " << std::to_string(selected_bands) << std::endl
+              << "  stay_on_initial_radio: " << std::to_string(stay_on_initial_radio) << std::endl
+              << "  stay_on_selected_device: " << std::to_string(stay_on_selected_device)
+              << std::endl;
```
Comment (adam1985d) [inline controller/src/beerocks/cli/beerocks_cli_bml.cpp]:
```suggestion
              << "  selected_bands: " << int(selected_bands) << std::endl
              << "  stay_on_initial_radio: " << int(stay_on_initial_radio) << std::endl
              << "  stay_on_selected_device: " << int(stay_on_selected_device)
              << std::endl;
```

**Review by adam1985d — APPROVED** [review]
looks good - gave small suggestions to the commit messages and other, but simple fixes so approving

File: controller/src/beerocks/bml/bml_defs.h (hunk 1 above)
Comment (itayx) [reply controller/src/beerocks/bml/bml_defs.h:602]: Changed

File: controller/src/beerocks/cli/beerocks_cli_bml.cpp (hunk 3 above)
Comment (itayx) [reply controller/src/beerocks/cli/beerocks_cli_bml.cpp]: Changed

File: controller/src/beerocks/cli/beerocks_cli_bml.cpp (hunk 2 above)
Comment (itayx) [reply controller/src/beerocks/cli/beerocks_cli_bml.cpp:566]: Changed
